src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/PInvokeModuleFixupNode.cs

```diff
@@ -45,6 +48,7 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
 
             builder.EmitZeroPointer();
             builder.EmitPointerReloc(nameSymbol);
+            builder.EmitInt((int)_dllImportSearchPath);
```

Comment:
Will we need to add more flags in the future? If so, coming up with a more compact enum (that is a contract used for storage in the file only) that can be extended to represent all the flags might be a good idea. People tend to do strictly additive changes when adding new features and I don't want us to end up with dozens of wasted bits in the format. The `DllImportSearchPath` has holes in it.